packages/pds-ember/.eslintrc.js

```diff
@@ -7,51 +7,52 @@ module.exports = {
     ecmaVersion: 2018,
     sourceType: 'module',
     ecmaFeatures: {
-      legacyDecorators: true
-    }
+      legacyDecorators: true,
+    },
   },
-  plugins: [
-    'ember'
-  ],
+  plugins: ['ember'],
   extends: [
     'eslint:recommended',
-    'plugin:ember/recommended'
+    'plugin:ember/recommended',
+    'plugin:prettier/recommended',
   ],
   env: {
-    browser: true
+    browser: true,
   },
   rules: {
-    'ember/no-jquery': 'error'
+    'ember/no-jquery': 'error',
+    'ember/no-empty-glimmer-component-classes': 0,
```

Comment:
No biggie and I'm only guessing what this is for here. Wondering if we could just remove the empty glimmer component classes? Or I suppose if they are needed for a context in the template, maybe adding default values for the properties on the context might help?

Reply:
_Probably_ but my memory is that there was a non-trivial number to address so it seemed worth punting on for now. I tried to get a more accurate number but for some reason I can no longer run `yarn lint:js` on this branch for reasons that I can't figure out right now 🤔 😭